### PR TITLE
Adding layouts in admin section

### PR DESCRIPTION
**Pull request by amit-webkul**

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 1]
```diff
@@ -42,6 +42,23 @@
         'false'           => 'False',
     ],
 
+    'components' => [
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:46]: Remove new line

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 2]
```diff
@@ -42,6 +42,23 @@
...
+        'layouts' => [
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:48]: Remove new line

File: packages/Webkul/Admin/src/Resources/views/components/layouts/sidebar/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                                <p class=" text-gray-600 px-[40px] py-[4px]">
+                                    @lang($subMenuItem['name'])
+                                </p>
+                            </a>
+                        @endforeach
+                    </div>
+                @endforeach
+            </div>
+        </nav>
+    </div>
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/components/layouts/sidebar/index.blade.php:41]: Add new line here

Comment (amit-webkul) [reply packages/Webkul/Admin/src/Resources/views/components/layouts/sidebar/index.blade.php:41]: Added

File: packages/Webkul/Admin/src/Resources/lang/en/app.php (hunk 2 above)
Comment (amit-webkul) [reply packages/Webkul/Admin/src/Resources/lang/en/app.php]: Done sir

File: packages/Webkul/Admin/src/Resources/lang/en/app.php (hunk 1 above)
Comment (amit-webkul) [reply packages/Webkul/Admin/src/Resources/lang/en/app.php]: Done

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**